wolfcrypt/src/aes_gcm_asm.asm

```diff
@@ -108,14 +108,24 @@ AES_GCM_encrypt PROC
         mov	rsi, rdx
         mov	r12, r8
         mov	rax, r9
-        mov	r8, QWORD PTR [rsp+96]
-        mov	r9d, DWORD PTR [rsp+104]
-        mov	r11d, DWORD PTR [rsp+112]
-        mov	ebx, DWORD PTR [rsp+120]
-        mov	r14d, DWORD PTR [rsp+128]
-        mov	r15, QWORD PTR [rsp+136]
-        mov	r10d, DWORD PTR [rsp+144]
-        sub	rsp, 160
+        mov	r8, QWORD PTR [rsp+256]
+        mov	r9d, DWORD PTR [rsp+264]
+        mov	r11d, DWORD PTR [rsp+272]
+        mov	ebx, DWORD PTR [rsp+280]
+        mov	r14d, DWORD PTR [rsp+288]
+        mov	r15, QWORD PTR [rsp+296]
+        mov	r10d, DWORD PTR [rsp+304]
+        vmovdqu	OWORD PTR [rsp+160], xmm6
```

Comment:
why AVX version instead of SSE (movdqu) ?

Reply:
Good point.
Generated code and it didn't know to produce SSE2 only code.
Fixed this.